build/lint

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
```

Comment:
why do we have a separate non-suffixed lint here?

Reply:
Good catch! Removed now.